src/js/components/state/overview/StateOverview.jsx

```diff
@@ -215,70 +218,115 @@ export default class StateOverview extends React.PureComponent {
-                            </h3>
-                            {tooltip}
-                            <table className="details__table">
-                                <tbody>
-                                    <tr>
-                                        <th>Population</th>
-                                        <td>{this.props.stateProfile.population} {populationSourceYear}</td>
-                                    </tr>
-                                    <tr>
-                                        <th>Awarded Amount Per Capita</th>
-                                        <td>{this.props.stateProfile.awardAmountPerCapita}</td>
-                                    </tr>
-                                    <tr>
-                                        <th>Median Household Income</th>
-                                        <td>{this.props.stateProfile.medianHouseholdIncome} {incomeSourceYear}</td>
-                                    </tr>
-                                </tbody>
-                            </table>
+                        <div className="state-section__viz totals-container">
+                            <div className="state-section__viz totals">
+                                <h3 className="state-overview__heading">
+                                Total Awarded Amount
+                                </h3>
+                                <div className="totals__amount">
+                                    {this.props.stateProfile.totalAmount}
+                                </div>
+                                <div className="totals__awards">
+                                from <span className="state-overview__total">{this.props.stateProfile.totalAwards}</span> prime awards
+                                </div>
+                                {(this.props.stateProfile.code) &&
+                                    <Link
+                                        className="state-section__award-button"
+                                        target="_blank"
+                                        rel="noopener noreferrer"
+                                        to="/search"
+                                        onClick={handleGoToAdvancedSearch}>
+                                        View awards to this state
+                                    </Link>
+                                }
+                            </div>
+                            <div className="state-section__viz loan">
+                                <FaceValueOfLoans amount={this.props.stateProfile.totalFaceValueLoanAmount} transactions={this.props.stateProfile.totalFaceValueLoanPrimeAwards} primeAwards heading="Face Value of Loans" headingClass="state-overview__heading" tooltipIcon="info" tooltipClasses="state-section__viz-loan__tt" tooltipComponent={stateOverviewLoanInfo} tooltipPosition="right" />
+                            </div>
                         </div>
                     </div>
```

Comment:
not sure if this branch is up-to-date with the latest from qat because i thought line 221 - 247 would have been removed in qat.  if merging in qat doesn't get rid if these, line 221 - 247 can be removed.